doc/doxygen/public/developer-faq.doxygen

```diff
@@ -0,0 +1,291 @@
+/**
+
+ @page developer_faq Developer FAQ
```

Comment:
let's use `\tableofcontents` for faq.



https://www.doxygen.nl/manual/commands.html#cmdtableofcontents

Reply:
I have....